lfroms/nimbus
# Don't pop last radar image for Environment Canada

### What is the problem being solved in this PR?
Previously, we `pop`-ed the last radar image returned from Environment Canada since it was not usually ready for display.  However, it seems like Environment Canada no longer returns the most recent image if it isn't ready yet, resulting in a 20-30 minute delay in radar imagery returned from Nimbus.

### Related issues or PRs
N/A

### How is this accomplished and why did you choose this approach?
By removing the line that removes the last timestamp.

### How to test changes
1. Launch Clouds and ensure that the last radar image is only 10-20 minutes delayed.

### Checklist
